panels/info/cc-info-panel.c

```diff
@@ -1469,7 +1471,12 @@ updates_apply_completed (GObject *object,
   if (error != NULL)
     {
       if (!g_error_matches (error, G_IO_ERROR, G_IO_ERROR_CANCELLED))
-        g_warning ("Failed to call Apply() on EOS updater: %s", error->message);
+        {
+          CcInfoPanel *self = user_data;
+          g_warning ("Failed to call Apply() on EOS updater: %s", error->message);
+          sync_state_from_updater (self, FALSE);
```

Comment:
Since a local error is pretty unlikely (assuming the system is properly set up, which we can assume), I don’t think it’s necessary to write the follow-up patch now. It’s probably a good idea to file a follow-up task about it though.